Source/SwiftLintFramework/Extensions/File+Cache.swift

```diff
@@ -68,7 +68,7 @@ private struct Cache<T> {
 extension File {
 
     private var cacheKey: String {
-        return path ?? "\(ObjectIdentifier(self).hashValue)"
+        return path ?? "\((contents as NSString).hash)"
```

Comment:
Oh, my bad! Thanks.
I also think we should use `contents` itself instead of their `hash`.